certbot-apache/certbot_apache/_internal/configurator.py

```diff
@@ -2306,20 +2215,14 @@ def _get_redirect_config_str(self, ssl_vhost: obj.VirtualHost) -> str:
         if ssl_vhost.aliases:
             serveralias = "ServerAlias " + " ".join(ssl_vhost.aliases)
 
-        rewrite_rule_args: List[str]
-        if self.get_version() >= (2, 3, 9):
-            rewrite_rule_args = constants.REWRITE_HTTPS_ARGS_WITH_END
-        else:
-            rewrite_rule_args = constants.REWRITE_HTTPS_ARGS
```

Comment:
I wonder if we shouldn't do more here with `constants.REWRITE_HTTPS_ARGS`. It's now only used to detect a redirect rule previously created by Certbot for Apache < 2.3.9. Since that's no longer supported, I wonder if we shouldn't:

1. Move the value of `constants.REWRITE_HTTPS_ARGS` into `constants.OLD_REWRITE_HTTPS_ARGS`.
2. Delete `constants.REWRITE_HTTPS_ARGS` and all references to it.
3. We could also rename `constants.REWRITE_HTTPS_ARGS_WITH_END` to `constants.REWRITE_HTTPS_ARGS` if we wanted, but I'm not sure that's necessary.

I think this has the effect of cleaning up the code a bit and having Certbot update the old redirect rule to the new format if the user is using Apache 2.4. What do you think?